packages/desktop-client/src/components/filters/FilterExpression.tsx

```diff
@@ -79,7 +79,8 @@ export function FilterExpression<T extends RuleConditionEntity>({
                 valueIsRaw={
                   op === 'contains' ||
                   op === 'matches' ||
-                  op === 'doesNotContain'
+                  op === 'doesNotContain' ||
+                  op === 'tags'
```

Comment:
nit: what are your thoughts on calling this `hasTags` instead? Thinking if we want to add a `doesNotHaveTags` in the future

Reply:
adopted your naming suggestion